### PR TITLE
With tests

### PR DESCRIPTION
**Pull request by ocefpaf**
# Not for merging!

This is the same as #188 but with the tests from https://github.com/ODM2/WOFpy/pull/156

I am opening this just to explore the alternatives and, once we settle, I'll implement them in #188

### CODE REVIEW

File: test/test_dao.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (lsetiawan) [inline test/test_dao.py]: This dao is for WaterML 1.0. Which is not our main focus currently when trying to solve this unicode problems. Though thanks for adding the similar test in anyways 😄

Comment (ocefpaf) [reply test/test_dao.py]: I picked your commit :smile:

**Comment by lsetiawan** [comment]
@ocefpaf could I close this?

**Comment by ocefpaf** [comment]
Sure! Just did ;-p

**Pull request closed without merging**